pkgs/development/python-modules/maestral/default.nix

```diff
@@ -44,8 +45,26 @@ buildPythonPackage rec {
     "--prefix" "PYTHONPATH" ":" "$out/lib/${python.libPrefix}/site-packages"
   ];
 
-  # no tests
-  doCheck = false;
+  checkInputs = [
+    pytestCheckHook
+    pytest-benchmark
+    pytest-rerunfailures
+  ];
+
+  disabledTests = [
+    # Tries to look at /usr
+    "test_filestatus"
+    # Requires systemd
+    "test_autostart"
+    # Network access
+    "test_check_for_updates"
+    # Tries to look at /usr/local/share
+    "test_path_exists_case_insensitive"
+    # Tried to look at /usr/local/share
```

Comment:
```suggestion
```

Reply:
Reshuffled the tests and comments a bit.